src/themes/teams/siteVariables.ts

```diff
@@ -6,19 +6,19 @@ import { pxToRem } from '../../lib'
 
 export const htmlFontSize = '14px' // what 1rem represents
 
-const blackRgbaFormat = (alpha: number): string => `rgba(37, 36, 36, ${alpha})`
-export const black = blackRgbaFormat(1)
+const blackRgbaFormat = (alpha: number): string => `rgba(37, 36, 35, ${alpha})`
+export const black = '#252423'
```

Comment:
Can we consolidate `black `and `fontBlack`?

Reply:
Done, I've also removed app-gray-12 since it will be removed in the near future anyway.